python/lsst/rubintv/analysis/service/client.py

```diff
@@ -0,0 +1,81 @@
+# This file is part of lsst_rubintv_analysis_service.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import logging
+
+import sqlalchemy
+import yaml
+from lsst.daf.butler import Butler
+from websocket import WebSocketApp
+
+from .command import DatabaseConnection, execute_command
+
+logger = logging.getLogger("lsst.rubintv.analysis.service.client")
+
+
+def on_error(ws: WebSocketApp, error: str) -> None:
+    """Error received from the server."""
+    print(f"\033[91mError: {error}\033[0m")
```

Comment:
Ok, I implemented this in a new `rubinTV_analysis_service.utils` module. I didn't do this before because the mock server (and even the print functions in the client module) aren't going to be used in production, they're just debugging tools as we develop the package, but I guess it doesn't hurt to include a`printc` method.